### PR TITLE
DS-9797- clarified conditions for importing external artifacts to the…

### PR DESCRIPTION
**Pull request by chtyler**
… s3 bucket for pipelines

## Description
I have updated the data science pipelines documentation to include information on importing artifacts generated outside of the pipeline in context. When you import external artifacts to your s3-compatible storage bucket, you can only import them to the s3 bucket that specified in the pipeline configuration. 

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: modules/configuring-a-pipeline-server.adoc [hunk 1]
```diff
@@ -68,6 +68,8 @@ The *Configure pipeline server* dialog appears.
 [IMPORTANT]
 ====
 If you specify incorrect data connection settings, you cannot update these settings on the same pipeline server. Therefore, you must delete the pipeline server and configure another one.
+
```
Comment (grainnejenningsRH) [inline modules/configuring-a-pipeline-server.adoc:71]: Suggest adding 'these' to this line to make it clear that 'you can only import artifacts to the S3...' applies only to the 2 cases you've defined. e.g.

_If you plan to import artifacts that were not generated by tasks in your pipeline, or you plan to use external artifacts that were not generated by any pipeline, you can only import **these** artifacts to the S3-compatible object storage bucket that you define in the *Bucket* field._

Same comment for modules/storing-data-with-data-science-pipelines.adoc.

Comment (chtyler) [reply modules/configuring-a-pipeline-server.adoc:71]: Resolved.

**Review by grainnejenningsRH — APPROVED** [review]

File: modules/configuring-a-pipeline-server.adoc [hunk 2]
```diff
@@ -68,6 +68,8 @@ The *Configure pipeline server* dialog appears.
...
+If you plan to import artifacts that were not generated by tasks in your pipeline, or you plan to use external artifacts that were not generated by any pipeline, you can only import these artifacts to the S3-compatible object storage bucket that you define in the *Bucket* field.
```
Comment (diegolovison) [inline modules/configuring-a-pipeline-server.adoc:72]: If you wish to use an existing artifact that was not generated by a task in the current pipeline you can use a [dsl.importer](https://kubeflow-pipelines.readthedocs.io/en/latest/source/dsl.html#kfp.dsl.importer) component to load the artifact from its URI.

Comment (chtyler) [reply modules/configuring-a-pipeline-server.adoc:72]: I can update this to mention the dsl importer component. However, we also need to mention that you can only import these artifacts to the S3-compatible object storage bucket that you define in the *Bucket* field in the pipeline configuration, as that was the point of the documentation request.

Comment (diegolovison) [reply modules/configuring-a-pipeline-server.adoc:72]: I see

```
If you wish to use an existing artifact that was not generated by a task in the current pipeline you can use a [dsl.importer](https://kubeflow-pipelines.readthedocs.io/en/latest/source/dsl.html#kfp.dsl.importer) component to load the artifact from its URI. You can only import these artifacts to the S3-compatible object storage bucket that you define in the *Bucket* field.
```

**Pull request merged**